GliaWidgets/Sources/CallVisualizer/CallVisualizer.swift

```diff
@@ -151,14 +153,17 @@ extension CallVisualizer {
 extension CallVisualizer {
     func startObservingInteractorEvents() {
         environment.interactorProviding()?.addObserver(self) { [weak self] event in
+            if case .stateChanged(.ended(.byOperator)) = event {
```

Comment:
Could you please move this statement to `Glia.startObservingInteractorEvents()`?
This is needed because there we also send `onEvent` actions to integrators. And since we log `.info("Engagement ended")` in Glia, we can change wording there to `"Call visualizer engagement ended"` (this one is more descriptive) and remove logging the same event from `CallVisualizer. startObservingInteractorEvents()`

What do you think?